pandas/tests/io/msgpack/test_sequnpack.py

```diff
@@ -50,13 +52,18 @@ def test_foobar_skip(self):
         unpacker.skip()
         assert unpacker.unpack() == ord(b'a')
         unpacker.skip()
-        pytest.raises(OutOfData, unpacker.unpack)
+        msg = "No more data to unpack"
+        with pytest.raises(OutOfData, match=msg):
+            unpacker.unpack()
 
     def test_maxbuffersize(self):
-        pytest.raises(ValueError, Unpacker, read_size=5, max_buffer_size=3)
+        msg = "read_size should be less or equal to max_buffer_size"
+        with pytest.raises(ValueError, match=msg):
+            Unpacker(read_size=5, max_buffer_size=3)
```

Comment:
I see at least two distinct tests in this one.  Let's at least break it up here where I've commented.